Domain/Sia.Shared/Extensions/Mediatr/ProxyHandler.cs

```diff
@@ -57,10 +58,18 @@ public abstract class ProxyHandlerBase<TRequest>
 
         protected virtual async Task<HttpResponseMessage> SendRequest(TRequest request, CancellationToken cancellationToken)
         {
-            var message = new HttpRequestMessage(Method(), RelativeUri(request));
-            AddContentToMessage(request, message);
-            await AddAuthorizationToMessage(request, message);
-            return await _client.SendAsync(message, cancellationToken);
+            try
+            {
+                var message = new HttpRequestMessage(Method(), RelativeUri(request));
+                AddContentToMessage(request, message);
+                await AddAuthorizationToMessage(request, message);
+                var result = await _client.SendAsync(message, cancellationToken);
+                return result;
+            }
+            catch (HttpRequestException ex)
```

Comment:
Does this exception get logged anywhere? If not yet, we should try to do so.

Reply:
we talked about putting logging in a subsequent PR.  want it to come in earlier?